### DISCUSSION TITLE
Pfsense with Suricata and filebeat?

### DISCUSSION THREAD

**Discussion by tbaror** [Security-Onion-Solutions/securityonion#5617] [Unsupported Versions] [answered]
Hello ,
I am currently using my own several Pfsense with Suricata listen on LAN interface , ingesting all events to central server using filebeat installed as service on each of the pfsense , ingest to logstash and to Elasticsearch , havig my own visualization with Kibana and Grafana,
I would like to evaluate SecurityOnion for our organization, my question is , is it possible to achieve same result with SecurityOnion  and gain all those feature with Hunt and alerts etc...?
Thanks

**Comment by n0th1nt0lOOz** [comment, marked as answer]
As #5582 answer. You can find the config for filebeat to make it usable in SO

**Comment by dougburks** [comment]
If you send the network traffic to Security Onion, then you can take advantage of not only Suricata, but also Zeek and full packet capture as well.
